### PR TITLE
Update formatter

### PR DESCRIPTION
**Pull request by buh**
Update formatter for wrap arguments.

### CODE REVIEW

File: Sources_v3/Database/DatabaseContainer_Tests.swift [hunk 1]
```diff
@@ -51,12 +51,14 @@ class DatabaseContainer_Tests: StressTestCase {
...
-                            },
-                            completion: $0)
+            container.write(
+                { session in
+                    try session.saveChannel(payload: self.dummyPayload(with: .unique), query: nil)
+                    try session.saveChannel(payload: self.dummyPayload(with: .unique), query: nil)
+                    try session.saveChannel(payload: self.dummyPayload(with: .unique), query: nil)
+                },
+                completion: $0
+            )
```
Comment (VojtaStavik) [inline Sources_v3/Database/DatabaseContainer_Tests.swift:61]: This finally looks good! 🎉

**Comment by VojtaStavik** [comment]
Let's wait with merging this after we close the open PRs

**Comment by buh** [comment]
@VojtaStavik please check PR
I reformat after rebase additionally 2 times.

**Comment by VojtaStavik** [comment]
@buh , OK do it then!

**Pull request merged**